filesystem.spec

```diff
@@ -144,15 +144,11 @@ posix.mkdir("/usr/lib/debug/usr/sbin")
 posix.mkdir("/usr/lib/debug/usr/lib")
 posix.mkdir("/usr/lib/debug/usr/%{_lib}")
 posix.mkdir("/usr/%{_lib}")
-posix.symlink("usr/bin", "/bin")
-posix.symlink("usr/sbin", "/sbin")
-posix.symlink("usr/lib", "/lib")
 posix.symlink("usr/bin", "/usr/lib/debug/bin")
```

Comment:
same here

Reply:
you should see all our changes from this 69091524bb8589efe2d129520b431fd14df26c51 commit

Reply:
updated to commit 404a52e1ed4ff56cd4e7aededdc810bbd0e43c62